### PR TITLE
Fixed tests build Error with Xcode 11.4

### PR DESCRIPTION
**Pull request by lennet**
## 📲 What

I removed the duplicated Import of SDWebImage and bumped the minimum version of swift-snapshot-testing to 1.7.2 (https://github.com/pointfreeco/swift-snapshot-testing/releases/tag/1.7.2)

## 🤔 Why

Compatibility with Xcode 11.4

### CODE REVIEW

**Comment by mltbnz** [comment]
The new PR template is already worth 💯 because of the why section

File: Settings.bundle/com.mono0926.LicensePlist.latest_result.txt [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: Crypto, nameSpecified: , owner: soffes, version: 0.6.0
 
 name: GrowingTextView, nameSpecified: , owner: KennethTsang, version: 0.7.2
 
-name: SDWebImage, nameSpecified: , owner: SDWebImage, version: 5.5.2
+name: SDWebImage, nameSpecified: , owner: SDWebImage, version: 5.4.1
```
Comment (fbernutz) [inline Settings.bundle/com.mono0926.LicensePlist.latest_result.txt]: Was this change intended?

Comment (lennet) [reply Settings.bundle/com.mono0926.LicensePlist.latest_result.txt]: Not not really. I tried to reset it but license-plist keeps regenerating it when I build the project

Comment (fbernutz) [reply Settings.bundle/com.mono0926.LicensePlist.latest_result.txt]: Maybe it helps to run the update dependency script locally?

Comment (fbernutz) [reply Settings.bundle/com.mono0926.LicensePlist.latest_result.txt]: Or have you tried to use the Xcode Update Swift Package Button? Mh..

Comment (lennet) [reply Settings.bundle/com.mono0926.LicensePlist.latest_result.txt]: I ended up with version 5.6 after manually updating

Comment (fbernutz) [reply Settings.bundle/com.mono0926.LicensePlist.latest_result.txt]: I think this is way better than downgrading 😄
